src/helpers/date-helpers.ts

```diff
@@ -45,3 +45,18 @@ export function getMonthName(month: number) {
 	}
 	return capitalizeFirstLetter(format(new Date(new Date().getFullYear(), month, 1, 0, 0, 0, 0), "MMMM", { locale: locale }));
 }
+
+export function titleForDateRange(intervalType: "Day" | "Week" | "Month", intervalStart: Date, variant?: "short" | "long") {
+	if (intervalType == "Month" && intervalStart.getDate() == 1) {
+		return `${getMonthName(intervalStart.getMonth())} ${intervalStart.getFullYear()}`;
+	}
+	else if (intervalType == "Week" || (intervalType == "Month" && intervalStart.getDate() != 1)) {
```

Comment:
The `&& intervalStart.getDate() != 1` is not needed here.  It will always be `true` if `intervalType == "Month"`.

Reply:
It will not always be true, you can have a month interval that starts on an arbitrary date

Reply:
(hmm this makes me realize that i'll need to adjust the date formats for international use cases at some point, since we're hardcoding ```"MM/dd/yyyy"``` here)

Reply:
It will always be `true` because if it were ever `false`, the first `if` condition would have already matched.

Reply:
What I mean is, if `intervalType == "Month"` and `intervalStart.getDate() != 1` is `false`, then `intervalStart.getDate() == 1` will be `true`, so we won't even get to this point in the code, because the first `if` condition was met.

Reply:
Ohhh I see what you mean - though you were saying this code path would never be reached.  You are saying it's redundant - yep, makes sense

Reply:
Fixed